cdap-app-fabric/src/main/java/co/cask/cdap/metadata/LineageAdmin.java

```diff
@@ -139,7 +140,16 @@ public Lineage computeLineage(final Id.Stream sourceStream, long startMillis, lo
   public Set<MetadataRecord> getMetadataForRun(Id.Run run) throws NotFoundException {
     entityExistenceVerifier.ensureExists(run.toEntityId());
 
-    Set<Id.NamespacedId> runEntities = new HashSet<>(lineageStoreReader.getEntitiesForRun(run));
+    // TODO: Remove this once lineageStoreReader stops using deprecated Id class
+    Set<NamespacedId> runEntities = new HashSet<>(
+      Collections2.transform(lineageStoreReader.getEntitiesForRun(run),
```

Comment:
to fix indentation, move ``lineageStoreReader.getEntitiesForRun(run),`` to the next line, and in-dent lines 147-150 by one more space.